libs/threading/tests/unit/CMakeLists.txt

```diff
@@ -9,19 +9,23 @@ set(threading_tests
     condition_variable2
     condition_variable3
     condition_variable4
+    condition_variable_race
```

Comment:
Just for consistency, let's maybe remove the "_" to `condition_variable_race` or add it to the `stop_tokenrace*`